### PR TITLE
chore(dashboard): batch of small fixes

### PR DESCRIPTION
**Pull request by LetItRock**
### What changed? Why was the change needed?

The batch of small improvements.

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/dashboard/src/components/workflow-editor/add-step-menu.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const MenuItem = ({
       <span className="text-xs">{children}</span>
       {disabled && (
         <Badge kind="pill" variant="soft" className="ml-auto opacity-40">
-          <BadgeContent variant="neutral">soon</BadgeContent>
+          <BadgeContent variant="neutral">coming soon</BadgeContent>
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/add-step-menu.tsx:56]: suggestion from: https://linear.app/novu/issue/NV-4667/[ux]-consider-replacing-the-soon-text-with-coming-soon-in-the-bubble

File: apps/dashboard/src/components/workflow-editor/configure-workflow.tsx [hunk 2]
```diff
@@ -67,6 +67,7 @@ export function ConfigureWorkflow() {
           <FormField
             control={control}
             name="name"
+            defaultValue=""
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/configure-workflow.tsx:70]: fixes the react warning about changing input from uncontrolled to controlled

File: apps/dashboard/src/components/workflow-editor/nodes.tsx [hunk 3]
```diff
@@ -235,7 +235,7 @@ export const AddNode = (_props: NodeProps<NodeType>) => {
   }
 
   return (
-    <div className="flex w-[300px] justify-center">
+    <div className="flex w-[300px] cursor-pointer justify-center">
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/nodes.tsx:238]: add step button cursor pointer fix: https://linear.app/novu/issue/NV-4684/workflow-overview-two-in-one-react-flow-and-cursor-on-hover

File: apps/dashboard/src/components/workflow-list.tsx [hunk 4]
```diff
@@ -102,7 +102,7 @@ export const WorkflowList = () => {
           ) : (
             <>
               {workflowsQuery.data.workflows.map((workflow) => (
-                <WorkflowRow workflow={workflow} />
+                <WorkflowRow key={workflow._id} workflow={workflow} />
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-list.tsx:105]: react missing key warning

**Review by SokratisVidros — APPROVED** [review]
📦

**Pull request merged**